kvs-jwt-verify/verify-jwt.js

```diff
@@ -1,16 +1,14 @@
 import crypto from 'crypto';
 import cf from 'cloudfront';
 
-// updated the original example from below URL to use KVS 
-// https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/example-function-validate-token.html
 
 //Response when JWT is not valid.
 const response401 = {
     statusCode: 401,
     statusDescription: 'Unauthorized'
 };
 
-// Replace the KVS_ID with your KVS ID
+// (Optional) Replace the KVS_ID with your KVS ID
 const kvsId = "KVS_ID";
```

Comment:
If this is not needed, we should just take this out. Making it optional is confusing (to me at least). Maybe just putting a comment in here about remembering to attach the KVS to your function before calling the `const kvsKey = 'jwt.secret'`